doc/source/train/user-guides/hyperparameter-optimization.rst

```diff
@@ -3,127 +3,168 @@
 Hyperparameter Tuning with Ray Tune
 ===================================
 
-Hyperparameter tuning with :ref:`Ray Tune <tune-main>` is natively supported with Ray Train.
+.. important::
+    This user guide shows how to integrate Ray Train and Ray Tune to tune over distributed hyperparameter runs
+    for the revamped Ray Train V2 available starting from Ray 2.43 by enabling the environment variable ``RAY_TRAIN_V2_ENABLED=1``.
+    **This user guide assumes that the environment variable has been enabled.**
 
+    Please see :ref:`here <train-tune-deprecation>` for information about the deprecation and migration.
 
-.. https://docs.google.com/drawings/d/1yMd12iMkyo6DGrFoET1TIlKfFnXX9dfh2u3GSdTz6W4/edit
 
-.. figure:: ../images/train-tuner.svg
+Ray Train can be used together with Ray Tune to do hyperparameter sweeps of distributed training runs.
+This is often useful when you want to do a small sweep over critical hyperparameters,
+before launching a run with the best performing hyperparameters on all available cluster resources for a long duration.
+
+Quickstart
+----------
+
+In the example below:
+
+* :class:`~ray.tune.Tuner` launches the tuning job, which runs a bunch of ``train_driver_fn`` trials with different hyperparameter configurations.
+* ``train_driver_fn``, which instantiates a ``TorchTrainer`` (or some other framework trainer), takes in a hyperparameter configuration and launches the distributed training job.
```

Comment:
nit/non-blocking: I personally find `train_driver_fn` a confusing name but not sure off the top of my head what a better name would be.

Reply:
do you like `launch_training_fn`

Reply:
I already use the term Train driver process in the text so it felt fine.